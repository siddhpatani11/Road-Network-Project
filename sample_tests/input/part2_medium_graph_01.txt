49 351
0 8 16 B
0 12 32 B
0 15 4 B
0 20 29 B
0 26 18 R
0 27 3 G
0 29 2 G
0 33 11 B
1 4 38 G
1 10 8 R
1 12 32 R
1 18 20 R
1 25 3 G
1 26 34 G
1 32 14 G
1 38 2 R
1 39 51 G
1 40 9 R
1 42 27 B
1 43 10 R
1 46 33 R
2 15 24 R
2 24 35 B
3 4 42 R
3 5 50 G
3 10 18 R
3 13 49 B
3 16 18 G
3 20 24 R
3 22 35 B
3 26 9 G
3 36 50 G
3 37 4 B
4 5 37 R
4 25 43 G
4 29 13 B
4 31 8 G
4 43 19 B
4 45 14 R
4 48 14 R
5 10 49 G
5 31 25 R
5 32 43 B
5 38 6 G
5 39 36 G
5 40 31 R
5 42 35 G
5 47 33 G
5 48 10 R
6 15 23 B
6 20 3 R
6 28 36 G
6 38 5 R
6 43 38 G
6 44 5 R
7 2 43 G
7 6 7 G
7 19 37 R
7 27 23 G
7 31 21 G
7 34 33 R
7 37 24 R
7 45 15 B
7 46 46 G
8 0 30 G
8 4 29 R
8 5 10 B
8 10 15 G
8 19 24 B
8 26 51 B
8 28 32 B
8 43 21 G
9 0 47 R
9 1 28 B
9 7 23 R
9 14 16 G
9 26 35 G
9 35 51 B
9 39 1 B
9 41 15 B
9 46 10 R
10 8 29 R
10 12 41 B
10 31 27 R
10 41 30 R
10 46 36 B
11 31 45 R
11 46 9 R
12 9 16 G
12 20 24 B
12 25 13 G
12 30 46 R
12 40 2 B
12 41 12 G
12 45 40 B
12 47 25 B
13 5 19 G
13 11 37 G
13 19 3 R
13 24 49 B
13 26 39 B
13 27 12 R
13 29 34 G
13 32 32 R
13 48 50 B
14 7 15 G
14 21 28 B
14 31 28 B
14 44 13 B
15 2 22 B
15 8 21 B
15 16 39 R
15 20 19 B
15 27 47 R
15 28 45 B
15 36 3 B
15 45 8 B
16 8 9 R
16 9 13 B
16 10 47 B
16 13 36 G
16 19 19 R
16 34 23 G
16 37 19 G
16 41 29 G
16 48 36 R
17 9 42 R
17 20 42 G
17 24 27 R
17 26 3 B
17 34 32 B
17 39 42 R
17 46 50 G
18 0 47 G
18 10 29 R
18 13 35 G
18 14 45 R
18 27 7 B
18 30 33 G
18 35 47 R
19 1 24 G
19 9 13 R
19 12 33 B
19 23 39 B
19 25 38 G
19 26 11 B
19 29 38 B
19 32 50 B
20 4 15 G
20 5 35 R
20 11 33 R
20 13 13 G
20 26 43 B
20 48 34 R
21 2 41 B
21 4 30 R
21 10 25 R
21 11 3 B
21 22 33 G
21 23 29 G
21 29 32 R
21 33 3 R
21 41 21 B
21 45 3 R
21 48 14 R
22 9 28 B
22 17 30 G
22 23 2 B
22 25 42 B
22 32 26 B
22 38 41 R
22 39 39 G
22 40 4 R
22 41 24 R
22 44 11 B
23 0 37 R
23 6 4 G
23 12 49 G
23 25 18 R
23 30 25 G
23 40 41 B
23 41 23 B
24 10 39 G
24 18 16 G
24 22 40 R
24 32 21 B
24 36 49 G
25 13 18 B
25 19 7 G
25 24 25 R
26 3 48 B
26 4 27 B
26 10 38 R
26 19 8 G
26 20 44 R
26 42 48 B
27 4 38 G
27 6 6 B
27 11 13 R
27 13 9 G
27 19 2 G
27 29 8 R
27 33 21 R
27 36 9 G
27 45 31 G
28 11 47 B
28 13 24 R
28 14 51 G
28 17 16 B
28 22 15 G
28 26 1 G
28 39 26 B
28 47 33 B
29 6 28 B
29 7 28 B
29 8 12 G
29 26 32 G
29 27 14 G
29 28 50 G
29 32 42 G
29 38 13 B
30 18 17 G
30 26 18 B
30 33 47 R
30 41 24 R
30 44 24 R
31 1 36 B
31 6 28 G
31 16 38 B
31 22 13 G
31 29 46 R
31 30 20 R
31 36 46 R
31 44 39 B
32 17 43 G
32 23 50 R
32 26 50 G
32 30 46 G
33 3 41 R
33 12 41 R
33 27 14 G
33 34 51 G
34 4 39 G
34 9 30 R
34 28 25 R
34 35 12 R
34 41 18 R
34 46 34 B
35 12 15 R
35 22 15 B
35 24 22 G
35 25 2 R
35 27 47 G
35 28 12 G
35 29 31 G
35 41 10 G
35 45 47 B
36 4 51 B
36 5 51 G
36 7 49 B
36 15 31 B
36 26 16 G
36 30 36 R
36 31 21 G
36 34 46 G
36 38 25 R
36 45 14 R
37 1 23 R
37 7 16 G
37 19 38 R
37 21 45 G
37 33 44 G
37 41 29 G
38 9 23 B
38 16 27 B
38 18 21 R
38 21 17 G
38 31 43 B
38 33 10 G
38 44 12 B
39 7 20 G
39 10 34 G
39 13 34 G
39 18 42 B
39 19 11 B
39 33 27 G
39 38 10 G
39 42 17 R
39 43 21 B
39 48 32 R
40 0 46 B
40 14 43 R
40 19 20 G
40 22 28 G
40 24 50 B
40 42 15 R
40 44 30 B
41 13 48 B
41 14 43 B
41 22 12 B
41 25 30 G
41 29 22 B
41 38 46 R
42 1 3 R
42 13 26 B
42 15 9 B
42 24 20 R
42 27 47 R
42 29 35 B
42 44 32 G
43 3 17 G
43 4 10 R
43 18 35 R
43 26 12 G
43 27 46 G
43 33 48 R
44 0 32 G
44 1 29 G
44 13 25 G
44 17 42 G
44 26 38 G
44 30 10 R
44 33 22 B
44 43 47 G
44 48 16 B
45 0 31 B
45 2 29 R
45 3 19 R
45 19 47 G
45 20 29 R
45 24 27 R
45 35 23 B
45 44 22 G
46 15 7 B
46 19 35 G
46 25 34 B
46 31 45 G
46 39 41 R
46 45 46 G
47 8 23 G
47 21 25 R
47 34 33 R
47 35 9 R
47 41 13 G
48 1 31 R
48 4 43 R
48 7 13 B
48 19 7 B
48 20 4 B
48 34 33 G
48 44 45 R
