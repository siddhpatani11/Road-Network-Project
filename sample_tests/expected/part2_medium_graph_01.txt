25 G 83 B 45 R 139 R 136 B 68 R 78 R 76 B 88 R 68 R 105 G 56 R 61 G 40 B 18 R 121 B 70 B 109 B 93 R 41 G 7 G 62 R 98 R 116 G 46 R 25 - 0 G 18 B 30 R 100 G 45 G 86 R 103 B 50 R 69 R 120 B 84 B 39 B 112 R 81 B 116 R 109 B 111 B 72 B 87 R 81 B 61 R 100 R 139 R 88 
